### ISSUE TITLE
Contentlanguage dropdown list  selection when only one is available

### ISSUE THREAD

**Issue by jarmosaarikko** [NatLibFi/Skosmos#743]
## At which URL did you encounter the problem?
http://finto.fi/ysa/en/?clang=fi&anylang=on

## What steps will reproduce the problem?
1.  going to the above address and selecting  Content language
2. If only one language is available, there should not show the selection "Aany language" because this implies that there would be other languages available.
3.  the SUBJECT of YSA is shown in Finnish, when selected "Any language" and interface is in English.
The English label is available  for this concept ins YSO.

## What is the expected output? What do you see instead?
"general concepts"    -  "yleiskäsitteet"

QUESTION:
Should Skosmos fetch the English label from YSO when the interface is English and Content language "Any language"

## What browser did you use? (eg. Firefox, Chrome, Safari, Internet explorer)
Chrome

**Comment by osma** [comment]
YSA and YSO labels are different things - even in Finnish they may be different. YSA is monolingual, there are no English labels there. It would be just confusing if Skosmos would fetch English labels for YSA concepts from YSO.

Regarding this:
> If only one language is available, there should not show the selection "Aany language" because this implies that there would be other languages available.

You are right, this is a bit confusing. There are cases where a vocabulary is configured for a single language (e.g. Finnish Corporate Names) while it actually contains some labels also in other languages. The "Any language" selection may be useful in those cases.

**Comment by jarmosaarikko** [comment]
You are correct. SEKO thesaurus is monolingual but has many labels in multiple languages, which are now (wrongly) coded as Finnish altLabel properties. If these would have the correct language codes there would be multiple language prefLabels available although SEKO is monolingual.

**Comment by osma** [comment]
Closing this as stale. Please reopen if still relevant.
